WordPress/src/main/res/values/strings.xml

```diff
@@ -3345,4 +3345,7 @@ translators: sample content for "Services" page template -->
     <string name="backup_item_roots">WordPress root (includes wp-config.php and any non WordPress files)</string>
     <string name="backup_item_contents">WP-content directory (excludes themes, plugins, and uploads)</string>
     <string name="backup_item_sqls">Site database (SQL)</string>
+
+    <string name="jetpack_icon_content_description">icon</string>
```

Comment:
Question (❓): Is that string all lowercase on purpose, maybe because it will not be going to be used anyway? Maybe `Jetpack icon` will be better?

Reply:
>Is that string all lowercase on purpose, maybe because it will not be going to be used anyway? Maybe Jetpack icon will be better?

It will be overwritten in the classes, but need in the XML as a placeholder. I thought of Jetpack icon - but that really doesn't describe the icon well either.

Reply:
True, I only suggested the `Jetpack icon` because of the resources being named as such! 😅 

But, as you said, it doesn't matter that much.